src/embed/hostEventClient/contracts.ts

```diff
@@ -0,0 +1,94 @@
+import { HostEvent } from '../../types';
+
+export enum UiPassthroughEvent {
```

Comment:
UiPassthroughEvent somehow doesn't read well. Can we rename this everywhere to be `UIPassthroughEvent` or `PassthroughHostEvent` which makes better context